svir/calculations/aggregate_loss_by_zone.py

```diff
@@ -70,7 +70,8 @@ def calculate_zonal_stats(loss_layer,
                           zone_id_in_zones_attr_name,
                           iface,
                           force_saga=False,
-                          force_fallback=False):
+                          force_fallback=False,
+                          sum_only=False):
     """
```

Comment:
Instead of a flag `sum_only=False` I would use a flag `extra=True`. It is more readable to see something like `if extra:..`
instead of `if not sum_only: ...` Notice that you always use the negative form.

Reply:
Done